woocommerce/woocommerce-gutenberg-products-block
# Implement `@wordpress/scripts` analyze script

Replaces `source-map-explorer` and the `npm run explore` command with the `@wordpress/scripts` equivilent. This can be ran with `npm run analyze-bundles` generating output like this:

![Screenshot 2022-01-18 at 13 33 29](https://user-images.githubusercontent.com/90977/149947064-b466ce1e-3c47-40bf-9c00-90c24a84203b.png)

[This was based on the work in Gutenberg here.](https://github.com/WordPress/gutenberg/pull/21827)

Fixes #5028

### Testing

How to test the changes in this Pull Request:

1. Run `npm install`
2. Run `npm run analyze-bundles`
3. Confirm the map is shown once build has completed.
4. Confirm no major build size increases (there will be a comment on the PR). This is because I modified the webpack config.
